src/features/movieList/index.js

```diff
@@ -1,9 +1,11 @@
 import { Content } from "./components/Content";
+import { Footer } from "./components/Footer";
 
 export const MovieListPage = () => {
-  return <Content />;
-};
-// napis
-// github
-// coś 
-// comment
\ No newline at end of file
+  return (
+    <>
+      <Content />
+      <Footer />
+    </>
+  );
+};
```

Comment:
These were very useful comments you've just removed and replaced with these ugly looking parenthesis hehe. obviously just messing. Nice you remembered what we have messed up during zoom meeting